vscode/src/extension.ts

```diff
@@ -514,16 +514,16 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
             vscode.window.showErrorMessage('Cannot find workspace path');
             return;
         }
+        
```

Comment:
We can maybe put this block of code in a try catch so that any error can be handled.